proto/trtl/v1/trtl.proto

```diff
@@ -130,6 +130,16 @@ message ServerStatus {
     string status = 1;
     string version = 2;
     string uptime = 3;
+    ReplicaStatus replica = 15;
+}
+
+message ReplicaStatus {
+    bool enabled = 1;
+    uint64 pid = 2;
+    string region = 3;
+    string name = 4;
+    string interval = 5;
+    string sigma = 6;
 }
```

Comment:
Added to have visibility into what the configuration is on k8s -- I can remove if you don't like it, but needed it for testing the deployment.

Reply:
Happy to have you leave this in; might be useful for configuration diagnostics/tuning in the future?